db/schema.rb

```diff
@@ -255,6 +255,29 @@
     t.index ["site_id"], name: "index_custom_user_fields_on_site_id"
   end
 
+  create_table "example_1_records", id: false, force: :cascade do |t|
```

Comment:
Does this file be commited? @entantoencuanto

Reply:
Yes, you shouldn't commit this file

Reply:
Yes, and there are 2 more tables in this file that shouldn't appear. Remember, if you have to upload changes to this file you must always do it in a separate commit. In this way, it would be enough to remove a commit instead of creating new commits or amending existing ones.

Reply:
Fixed